### ISSUE TITLE
Feedback User 273: add drag & drop functionality to feedback form 

### ISSUE THREAD

**Issue by sara-georgy** [stadtnavi/digitransit-ui#675]
**For future improvements:** The functionality drag & drop should be added to the feedback form in order to upload attachments and screenshots easier.

**Comment by MartinH-open** [comment]
Yes, that is my feedback. Thanks. Here in Github issues it is really easy to drag and drop pictures into the submit description.
It makes is easy to submit many issues , while on web-ui I hesitate providing feedback with images because it requires multiple steps on my side.
